### ISSUE TITLE
DaySelectedBackgroundColor - Circle not Rendering

### ISSUE THREAD

**Issue by johnyenter-briars** [ME-MarvinE/XCalendar#99]
**Describe the bug**

Hi there! 

Thanks for all the work you put into `XCalendar`!

I think I might be experiencing a bug in setting the color value for the dot that indicates the currently selected date.

Looking at the [Sample Code]() it looks to me like it should be set on the `cx:DayView` - where you assign it to a `Color` value.

I've done that like so:

```
<xc:DayView
    ...
    SelectedBackgroundColor="{StaticResource DaySelectedBackgroundColor}"
    ...
    >
</xc:DayView>
```

And then set the resource in `App.xaml` like

```
<Application.Resources>
    ...
    <Color x:Key="DaySelectedBackgroundColor">Green</Color>
    ...
</Application.Resources>
```

However, the selected indicator still does not populate. Screenshot: 
![xcalendarissue](https://user-images.githubusercontent.com/42982326/212799622-43e9f4b8-64f7-43d2-bb66-6ed819c25be1.jpg)

Are there other properties that I need to set to enable this feature?

**Expected behavior**

I would think there should be a green dot when selecting on a day. 

**Steps to reproduce OR link to code**

I've uploaded my current code to [this repo / branch](https://github.com/johnyenter-briars/cal/blob/dev/bring_in_xcalendar/CAL/Views/CalendarPage.xaml)

**Xamarin Forms or .NET MAUI (If related to UI)**

.NET MAUI

**Additional context (Optional)**

**Device Info (Optional)**
Device Model: samsung galaxy s20 FE
Android Version: 12, One UI version 4.1
IOS Version:
Windows Version:

**Comment by ME-MarvinE** [comment]
When you say "Indicators", do you mean like on of the two red dots in your image? If so, there is no built in functionality for indicators. You will need to implement the logic yourself.

If you mean an actual day, the DayView has some properties you need to set in order to get the state-based properties working in different states. The [Displaying Dates](https://github.com/ME-MarvinE/XCalendar/wiki/Displaying-Dates) page lists them in the 'CalendarView' section.

**Comment by ME-MarvinE** [comment]
When you say "Indicators", do you mean like on of the two red dots in your image? If so, there is no built in functionality for indicators. You will need to implement the logic yourself.

If you mean an actual day, the DayView has some properties you need to set in order to get the state-based properties working in different states. The [Displaying Dates](https://github.com/ME-MarvinE/XCalendar/wiki/Displaying-Dates) page lists them in the 'CalendarView' section. Usually you would just bind them to the properties of the same name from the binding context.

**Comment by ME-MarvinE** [comment]
Also, just to confirm, is it the 'master' branch you wanted to link to for the reproduction of the issue?

**Comment by johnyenter-briars** [comment]
Hey @ME-MarvinE  - thanks for looking into this!

I'm pretty sure I linked to the exact branch / file in the ticket - but here is the exact full link: https://github.com/johnyenter-briars/cal/blob/dev/bring_in_xcalendar/CAL/Views/CalendarPage.xaml

The UI i'm referring to is the little dot that represents which dates are currently selected:
![xcalendarexample](https://user-images.githubusercontent.com/42982326/212921698-9d42d280-6d55-4879-afd5-37ee3c01dc18.jpg)

**Comment by ME-MarvinE** [comment]
@johnyenter-briars I was opening the link through the Mobile app which kept leading me to the 'master' branch. The links work fine on PC.

Eventually I tried using a Calendar with `CalendarDay` instead of `EventDay` and it worked. 

The `EventDay` and `Event` classes in the sample app (along with other classes) use `PropertyChanged.Fody` to inject usage of the `INotifyPropertyChanged` interface into the shorthand setter ({ get; set; }) by adding the     `AddINotifyPropertyChangedInterface` attribute to the 'BaseObservable' classes. 

This package nor the attribute is present in the project and there was no replacement for its functionality. You will need to either implement the `INotifyPropertyChagedInterface' manually, or install the `PropertyChanged.Fody` nuget package and re-add the `AddINotifyPropertyChangedInterface` attribute wherever it was removed.

I implemented the interface manually into the `EventDay` class and everything seemed to work correctly.
